project/Build.scala

```diff
@@ -573,7 +601,6 @@ object Build extends sbt.Build {
           normalizedName := "sbt-scalajs",
           name in bintray := "sbt-scalajs-plugin", // "sbt-scalajs" was taken
           sbtPlugin := true,
-          scalaVersion := "2.10.6",
```

Comment:
Ok. Fair enough. And its not like anyone compiles it from sbt anyways... just `reload` :P